Algorithms/Dynamic Programming/rod_cutting(DP).cpp

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
Please provide the link to the question in the comment form.

Reply:
Done in new commit please review the changes